diffrax/_brownian/tree.py

```diff
@@ -63,6 +64,25 @@
 _BrownianReturn = TypeVar("_BrownianReturn", bound=AbstractBrownianIncrement)
 
 
+# An internal dataclass that holds the rescaled Levy areas
+# in addition to the non-rescaled ones, for the purposes of
+# taking the difference between two Levy areas.
+class _LevyVal(eqx.Module):
+    dt: RealScalarLike
+    W: Array
+    H: Optional[Array]
+    bar_H: Optional[Array]
+    K: Optional[Array]
+    bar_K: Optional[Array]
+
+    def __post_init__(self):
```

Comment:
Nit: prefer `__check_init__` where possible. This is an Equinox-specific extension that (a) runs even when you inherit, and (b) isn't ignored if you define a custom `__init__` method, and (c) doesn't allow you to still mutate the class whilst it is running.

Reply:
Oh, I understand, will do.